main.go

```diff
@@ -1072,7 +1072,8 @@ func findFATMounts(options *compileopts.Options) ([]mountPoint, error) {
 				continue
 			}
 			fstype := fields[2]
-			if fstype != "vfat" {
+			// chromeos bind mounts use 9p
+			if fstype != "vfat" && fstype != "9p" {
```

Comment:
```suggestion
			if !(fstype == "vfat" || fstype == "9p") {
```

Nitpick: perhaps more maintainable to stick to demorgan's law?

Reply:
Sure. Do you prefer a rebase + force push it second commit?

Reply:
rebase + force push, please

Reply:
done